tests/test_model.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+
+class TestManufacturer(TestCase):
+    def setUp(self):
+        self.name = "Test Manufacturer"
+        self.country = "Test Country"
+        self.manufacturer = Manufacturer.objects.create(
+            name=self.name,
+            country=self.country,
+        )
+
+    def test_manufacturer_str(self):
+        self.assertEqual(str(self.manufacturer), f"{self.name} {self.country}")

```

Comment:
Ensure that the `__str__` method in the `Manufacturer` model returns the string in the format `{name} {country}`. If the actual implementation differs, adjust the test accordingly.